draft-ietf-jsonpath-base.md

```diff
@@ -517,6 +521,18 @@ This is the concatenation of three lists, two of length one containing `0`, `1`,
 As a consequence of this approach, if any of the selectors selects no nodes,
 then the whole JSONPath selects no nodes.
 
+The current item selector `@` when used outside a filter behaves as the document
+root selector unless a child node is given to the evaluator.
+When a child node is used as current item, evaluator must select it as starting
+node instead of the document root when an expression starting with `@` is
+evaluated.
+
+When a child node is used as current item, the current item selector `@` selects
+it as starting node instead of the document root.
```

Comment:
Usage of "node" here?  I think this is one we've decided to avoid.

Reply:
I don't know who "we" is; I think we are still discussing terminology.

Reply:
(I'm not so sure about "child node", the term "child" is usually used for direct descendants.)

Reply:
Yes, we're still discussing it, which means that it's something to avoid for now.

Reply:
ok, I can replace `node` with `item` and `child node` with `subdocument`. I think that further improvements might be applied on future PRs.

Reply:
Please wait for PR#72 to settle.

Reply:
"document" certainly is not the terminology we want to use.

Reply:
I think that I can replace "`The current item selector`" ... "`with @ is evaluated.`"

with

```
The current item selector `@` can be used to represent the current item being
evaluated. The initial current item defaults to the document root unless a
subdocument is given to the evaluator as initial current item.
```

I think I can safely remove "`When a child node is used as current item, evaluator must select it as starting
node instead of the document root when an expression starting with @ is
evaluated.`" and  "`When a child node is used as current item, the current item selector @ selects
it as starting node instead of the document root.`" since the proposed contains the same information but with a better phrasing.

@gregsdennis Let me know if it sounds good to you so I can push changes I described here.

Reply:
I think @cabo makes a good point about resolving it in another issue when we finish discussing "node."

No worries.

Reply:
@gregsdennis: Ok, I'm keeping node for now, we can improve this with a future PR.

@cabo I removed document and replaced it again with "root node". I also replaced "child node" with "descendant node" which is more accurate.

So I will commit & push:
```
The current item selector `@` can be used to represent the current item being
evaluated. The initial current item defaults to the root node unless a
descendant node is given to the evaluator as initial current item.
```

Let me know if you have any further feedback, otherwise let me know you are ok for now so I can push it.